### ISSUE TITLE
Representable deprecation warnings

### ISSUE THREAD

**Issue by tsaleh** [google/google-api-ruby-client#407]
I'm getting tons of these:

```
[Representable] Mixing user and representable options is deprecated. Please provide your options via :user_options.
[Representable] Positional arguments for `:if` are deprecated. Please use options or keyword arguments.
  if: ->(options) { options[:user_options] } or
  if: ->(user_options:) {  }
  Learn more here: http://trailblazerb.org/gems/representable/upgrading-guide.html#positional-arguments
```

Here's my `Gemfile.lock`:

```
GEM
  remote: https://rubygems.org/
  specs:
    activesupport (4.2.6)
      i18n (~> 0.7)
      json (~> 1.7, >= 1.7.7)
      minitest (~> 5.1)
      thread_safe (~> 0.3, >= 0.3.4)
      tzinfo (~> 1.1)
    addressable (2.4.0)
    declarative (0.0.7)
      uber (>= 0.0.15)
    faraday (0.9.2)
      multipart-post (>= 1.2, < 3)
    google-api-client (0.9.pre3)
      activesupport (>= 3.2)
      addressable (~> 2.3)
      googleauth (~> 0.2)
      hurley (~> 0.1)
      memoist (~> 0.11)
      mime-types (>= 1.6)
      multi_json (~> 1.11)
      representable (~> 2.1)
      retriable (~> 2.0)
      thor (~> 0.19)
    googleauth (0.5.1)
      faraday (~> 0.9)
      jwt (~> 1.4)
      logging (~> 2.0)
      memoist (~> 0.12)
      multi_json (~> 1.11)
      os (~> 0.9)
      signet (~> 0.7)
    hurley (0.2)
    i18n (0.7.0)
    json (1.8.3)
    jwt (1.5.4)
    launchy (2.4.3)
      addressable (~> 2.3)
    little-plugger (1.1.4)
    logging (2.1.0)
      little-plugger (~> 1.1)
      multi_json (~> 1.10)
    memoist (0.14.0)
    mime-types (3.0)
      mime-types-data (~> 3.2015)
    mime-types-data (3.2016.0221)
    minitest (5.8.4)
    multi_json (1.11.3)
    multipart-post (2.0.0)
    os (0.9.6)
    representable (2.4.1)
      declarative (~> 0.0.5)
      uber (~> 0.0.15)
    retriable (2.1.0)
    signet (0.7.2)
      addressable (~> 2.3)
      faraday (~> 0.9)
      jwt (~> 1.5)
      multi_json (~> 1.10)
    thor (0.19.1)
    thread_safe (0.3.5)
    tzinfo (1.2.2)
      thread_safe (~> 0.1)
    uber (0.0.15)

PLATFORMS
  ruby

DEPENDENCIES
  google-api-client (= 0.9.pre3)
  googleauth
  launchy
  thor

BUNDLED WITH
   1.11.2
```

**Comment by sqrrrl** [comment]
Will update in the 0.10 release.

**Comment by Rio517** [comment]
I noticed this was closed, but would love to know the status update. Was a fix added to 0.10?  Any chance it made it into 0.9.14?

**Comment by sqrrrl** [comment]
It'll be in 0.10. Mostly done with it, just finishing up some work with repackaging the generated APIs into separate gems and some other improvements to make it easier to keep the generated APIs up to date.

I don't have an ETA, but I hope to have it all wrapped up in the next couple of weeks if not sooner.
